frameworks/g-cloud-9/questions/services/lotName.yml

```diff
@@ -0,0 +1,2 @@
+question: 'Product type'
+type: text
```

Comment:
It may also be used for indexing G-Cloud services into Elasticsearch, so that lot names can be shown in search result summaries, eg.:
![screen shot 2017-01-30 at 13 26 16](https://cloud.githubusercontent.com/assets/6525554/22424752/d6956f60-e6ef-11e6-84f6-a27b1491076a.png)